client/src/views/Results.vue

```diff
@@ -104,6 +115,22 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
+.top-left-controls {
+    display: flex;
+    flex-direction: column;
+    align-items: start;
```

Comment:
This should be `align-items: flex-start`, because `start` is not supported by all browsers. Does the same thing